### DISCUSSION TITLE
Home tab

### DISCUSSION THREAD

**Discussion by hme3** [sumatrapdfreader/sumatrapdf#4275] [Documentation]
I like how you can now close home tab if other tabs are open. Is there a way to open it again without closing all tabs?

**Comment by kjk** [comment]
No, there is not way to do that.

**Comment by hme3** [comment]
I did find if you have NoHomeTab = false, have tabs open, and close home tab, and close and re-open SumatraPDF home tab comes back

**Comment by proideo** [comment]
I think it would be nice to be able to open the Home Tab with a shortcut for quick access to recent files without having it open by default.

**Reply by GitHubRulesOK** [reply]
If home is closed simply open new window but to list recent use CTRL + K
![image](https://github.com/user-attachments/assets/642a4041-fdbc-4be1-953b-d4d43a98a91d)
![image](https://github.com/user-attachments/assets/c27f135d-63ce-4cf2-8ab7-ae745915ff47)

**Reply by proideo** [reply]
Ok. Opening a new window is a workaround to open recent files visually but it would be perfect to open Home in a tab of the same window.

**Reply by GitHubRulesOK** [reply]
Well if it is not possible to restore one that has been closed
https://github.com/sumatrapdfreader/sumatrapdf/discussions/4275#discussioncomment-9558314

Then the simplest answer has to be dont close it.
![image](https://github.com/user-attachments/assets/0814265b-ec96-4e21-b5c4-d4cadc6fab2e)
